util/stats/timer.go

```diff
@@ -45,6 +45,11 @@ func (t *Timer) ElapsedTime() time.Duration {
 	return time.Since(t.start)
 }
 
+// Duration returns the duration value of the timer (in ns).
```

Comment:
It returns it as a `time.Duration`, which has no unit per se, so omit the `(in ns)`.